Modules/xxmodule.c

```diff
@@ -9,8 +9,8 @@
    You will probably want to delete all references to 'x_attr' and add
    your own types of attributes instead.  Maybe you want to name your
    local variables other than 'self'.  If your object type is needed in
-   other files, you'll have to create a file "foobarobject.h"; see
-   floatobject.h for an example. */
+   other files, you'll have to create a separate header file for it in the
+   Include/ directory. */
```

Comment:
```suggestion
   other files, you'll have to create a separate header file for it. */
```

The location of the header file depends on the user project so no need to say where to place it (`Include` here is used by CPython but you can also have your own `include` folder)